### ISSUE TITLE
Remove liveCourseAccess concept from StoryblokCoursePage

### ISSUE THREAD

**Issue by eleanorreem** [chaynHQ/bloom-frontend#1178]
### Overview
We used to have a concept of a "Live course".  This was a course that was being delivered live rather than being available online for everyone at any time. This concept has been retired and we want to remove it from the code to simply. This ticket is to remove the concept of liveCourse from the StoryblokCoursePage component.

### Action Items

- [ ] Remove liveCourseAccess concept from the StoryblokCoursePage component. This includes courseLiveNow, courseComingSoon or courseLiveSoon concept.
- [ ] Check everything looks right on course page where the component is rendered
- [ ] Run cypress tests to ensure that everything is working.

**Comment by kyleecodes** [comment]
Correct. Thank you for pointing out. The issue was divided for components and Storyblok components. I'll assign you since you completed it, link in PR, then close it.

**Comment by kyleecodes** [comment]
Completed: https://github.com/chaynHQ/bloom-frontend/pull/1271
